packages/common/utility-types/src/arrayElements.en.md

```diff
@@ -0,0 +1,3 @@
+# arrayElements
```

Comment:
According to [other TypeScript built-in types](https://www.typescriptlang.org/docs/handbook/utility-types.html) I guess names like `ElementType<>` follows the convention. What do you think?

Reply:
Yes I think type or interface should be Pascal case😁

Reply:
The name ElementType is more intuitive